schema.sql

```diff
@@ -1,16 +1,33 @@
--- Active: 1695416737359@@127.0.0.1@5432@vet_clinic
-/* Database schema to keep the structure of entire database. */
-
-CREATE TABLE animals (
-    id SERIAL PRIMARY KEY,
-    name VARCHAR(255),
-    date_of_birth DATE,
-    escape_attempts INT,
-    neutered BOOLEAN,
-    weight_kg DECIMAL(10, 2)
+-- vetclinic database
+
+CREATE DATABASE vet_clinic;
+
+-- animals table
+
+CREATE TABLE animals ( id SERIAL PRIMARY KEY,
+                                         name VARCHAR(255),
+                                              date_of_birth DATE, escape_attempts INTEGER, neutered BOOLEAN, weight_kg DECIMAL, species VARCHAR(255), -- 'species column'
+ species_id INTEGER, -- 'species_id column'
+ owner_id INTEGER -- 'owner_id column'
 );
 
--- Modify your schema.sql file.
---Add a column species of type string to your animals table.
-ALTER TABLE animals
-ADD COLUMN species VARCHAR(255)
\ No newline at end of file
+-- foreign key
```

Comment:
:red_circle: [REQUIRED] Please make sure you delete the column species from the animal table as requested here :arrow_down: 
![Screenshot from 2023-09-23 13-11-11](https://github.com/Errytagedesign/vet_clinic/assets/26736582/7acdc9a4-13da-4be2-8bcf-5b473a0627b7)
You can use this query :arrow_down: 
` ALTER TABLE animals DROP column species;`